lib/evaluate.js

```diff
@@ -179,6 +183,11 @@ export const evaluate = async ({
   }
   const setScoresDuration = Date.now() - start
 
+  ;(async () => {
+    await tx.wait()
+    cancelStuckTransactions.successful(tx)
+  })().catch(console.error)
```

Comment:
Since you are already using `.catch`, how about this:

```js
  // run in the background
  tx.wait()
    .then(() => cancelStuckTransactions.successful(tx))
    .catch(console.error)
```

Reply:
I'm using the wrapper with catch here because it's a pattern:

```js
;(async () => {
  // everything in here happens outside the flow
}).catch(() => {
  // we always need error handling with this pattern
})
```

I try to avoid `Promise#then()` and `Promise#catch()` as much as possible, because they can be seen as lower-level building blocks that should be replaced by `async/await` and `try/catch` as much as possible